toolchain/sem_ir/builtin_function_kind.cpp

```diff
@@ -123,10 +136,38 @@ using IntT = TypeParam<0, AnyInt>;
 // Not a builtin function.
 constexpr BuiltinInfo None = {"", nullptr};
 
+// "int.negate": integer negation.
+constexpr BuiltinInfo IntNegate = {"int.negate",
+                                   ValidateSignature<auto(IntT)->IntT>};
+
 // "int.add": integer addition.
 constexpr BuiltinInfo IntAdd = {"int.add",
                                 ValidateSignature<auto(IntT, IntT)->IntT>};
 
+// "int.sub": integer subtraction.
+constexpr BuiltinInfo IntSub = {"int.sub",
+                                ValidateSignature<auto(IntT, IntT)->IntT>};
+
+// "int.mul": integer multiplication.
+constexpr BuiltinInfo IntMul = {"int.mul",
+                                ValidateSignature<auto(IntT, IntT)->IntT>};
```

Comment:
You might consider something like "BinaryIntToInt", "BinaryIntToBool", but this seems fine too.